otx/algorithms/detection/tasks/nncf.py

```diff
@@ -323,8 +327,7 @@ def optimize(
             loading_stage_progress_percentage=5,
             initialization_stage_progress_percentage=5,
         )
-        learning_curves = DefaultDict(OTXLoggerHook.Curve)  # type: DefaultDict
```

Comment:
Not used?

Reply:
This log_config is added in `_initialized` method in [here](https://github.com/openvinotoolkit/training_extensions/blob/feature/otx/otx/algorithms/common/tasks/training_base.py#L231).
It is a duplicate addition.